torch/_dynamo/output_graph.py

```diff
@@ -219,7 +220,14 @@ def wrap_name(module_key):
 
             def wrap_name(module_key):
                 return NNModuleVariable(type(mod), module_key, **options)
-
+        elif isinstance(mod, torch.SymIntNode):
```

Comment:
huh, what, we're storing SymInts on a module? That's weird

Reply:
No, this is not a module, at all. Let me change the type annotations, it's dead wrong.